config/devices_test.go

```diff
@@ -338,6 +338,11 @@ func TestDevicesConfig_LoadFromFile(t *testing.T) {
 							t.Errorf("expected %v got %v", *expected.DefaultValue, *result.DefaultValue)
 						}
 					}
+					if expected.MaxSegments != nil {
+						if result.MaxSegments == nil || *expected.MaxSegments != *result.MaxSegments {
+							t.Errorf("expected %v got %v", *expected.MaxSegments, *result.MaxSegments)
```

Comment:
Nitpick: Since this is just a test, but this would panic w/ a nil pointer dereference if `result.MaxSegments == nil`.